projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php

```diff
@@ -184,6 +198,13 @@ class_exists( 'Jetpack_AMP_Support' )
 			// Override Notification module to include RTL styles.
 			add_action( 'a8c_wpcom_masterbar_enqueue_rtl_notification_styles', '__return_true' );
 		}
+
+		// Hides and replaces the language dropdown for the current user, on Atomic.
+		if ( jetpack_is_atomic_site() &&
+			empty( $_GET['user_id'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			add_action( 'user_edit_form_tag', array( $this, 'hide_language_dropdown' ) );
+			add_action( 'personal_options', array( $this, 'replace_language_dropdown' ) );
```

Comment:
What if other plugins hook into `personal_options` and the callbacks run before this? That may explain why I'm not seeing the dropdown replaced (seems that the AMP plugin installed on my Atomic site is adding some extra personal options).

Reply:
I tried deactivating the AMP plugin and I do see the dropdown replaced now, so it seems there is some conflict here.

With AMP | Without AMP
--- | ---
<img width="898" alt="with amp" src="https://user-images.githubusercontent.com/1233880/117932838-2a3e5780-b301-11eb-951b-680fdfa22920.png"> | <img width="731" alt="without amp" src="https://user-images.githubusercontent.com/1233880/117932846-2c081b00-b301-11eb-8b3c-6c02013274fc.png">

Reply:
We're fixing the AMP conflict separately: https://github.com/Automattic/jetpack/pull/19828